### PR TITLE
survey view

### LINKED ISSUE #63: Create survay view results
create a new Responce object(DTO) :

[
{
id: 1,
answers :[
fish : 50,
chicken: 20
]
total: 70
}

]

### PR DESCRIPTION
**Pull request by BtsuanEyakem**

### CODE REVIEW

File: src/main/java/edu/miu/cs545/project/model/entity/SurveyView.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+@Entity
+@Setter
+@Getter
+@EqualsAndHashCode(callSuper = true)
+public class SurveyView extends BasicEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Setter(AccessLevel.NONE)
+    private Long id;
```
Comment (darkhanzhursin) [inline src/main/java/edu/miu/cs545/project/model/entity/SurveyView.java]: no need id here, it is extended from parent

File: src/main/java/edu/miu/cs545/project/model/entity/SurveyQuestionView.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@Entity
+public class SurveyQuestionView {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```
Comment (darkhanzhursin) [inline src/main/java/edu/miu/cs545/project/model/entity/SurveyQuestionView.java]: no need id here, it is extended from parent

File: src/main/java/edu/miu/cs545/project/model/entity/SurveyChoiceVotes.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+@Data
+@Entity
+public class SurveyChoiceVotes {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```
Comment (darkhanzhursin) [inline src/main/java/edu/miu/cs545/project/model/entity/SurveyChoiceVotes.java]: no need id here, it is extended from parent

**Review by darkhanzhursin — CHANGES_REQUESTED** [review]

File: src/main/java/edu/miu/cs545/project/service/impl/SurveyViewServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
+package edu.miu.cs545.project.service.impl;
+
```
Comment (nabil-miu) [inline src/main/java/edu/miu/cs545/project/service/impl/SurveyViewServiceImpl.java:2]: remove extra blank lines

**Pull request closed without merging**